### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by CaroGo**
I did a few changes...

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -275,8 +275,8 @@
     <string name="occlusion">Oclusión</string>
     <string name="overview_bolusprogress_stop">Detener</string>
     <string name="overview_bolusprogress_stoppressed">Parar pulsado</string>
-    <string name="waitingforpumpclicktorefresh">Esperando bomba. Click to refresh.</string>
-    <string formatted="false" name="overview_bolusprogress_goingtodeliver">Va a entregar% .2fU</string>
+    <string name="waitingforpumpclicktorefresh">Esperando bomba. Pulsar para actualizar.</string>
+    <string formatted="false" name="overview_bolusprogress_goingtodeliver">Se entregará % .2fU</string>
```
Comment (AdrianLxM) [inline app/src/main/res/values-es/strings.xml]: entregar o suministrar?

**Comment by CaroGo** [comment]
It is more or less the same.... I prefer "entregar" because it is more common! O en español.... Prefiero entregar por ser más común... Pero en realidad da igual! 😉
BgCaro
-------- Ursprüngliche Nachricht --------Von: AdrianLxM <notifications@github.com> Datum: 31.01.18  20:19  (GMT+01:00) An: MilosKozak/AndroidAPS <AndroidAPS@noreply.github.com> Cc: CaroGo <salog1512@gmail.com>, Author <author@noreply.github.com> Betreff: Re: [MilosKozak/AndroidAPS] Update strings.xml (#633) 
@AdrianLxM commented on this pull request.



In app/src/main/res/values-es/strings.xml:
> @@ -275,8 +275,8 @@
     <string name="occlusion">Oclusión</string>
     <string name="overview_bolusprogress_stop">Detener</string>
     <string name="overview_bolusprogress_stoppressed">Parar pulsado</string>
-    <string name="waitingforpumpclicktorefresh">Esperando bomba. Click to refresh.</string>
-    <string formatted="false" name="overview_bolusprogress_goingtodeliver">Va a entregar% .2fU</string>
+    <string name="waitingforpumpclicktorefresh">Esperando bomba. Pulsar para actualizar.</string>
+    <string formatted="false" name="overview_bolusprogress_goingtodeliver">Se entregará % .2fU</string>

entregar o suministrar?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/MilosKozak/AndroidAPS","title":"MilosKozak/AndroidAPS","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/MilosKozak/AndroidAPS"}},"updates":{"snippets":[{"icon":"PERSON","message":"@AdrianLxM commented on #633"}],"action":{"name":"View Pull Request","url":"https://github.com/MilosKozak/AndroidAPS/pull/633#pullrequestreview-93049129"}}}

**Comment by MilosKozak** [comment]
ready to merge?

**Comment by CaroGo** [comment]
Yes, I guess! If there are no conflics...?

**Comment by AdrianLxM** [comment]
@CaroGo could you please target that PR to `dev`, not `master`. Thank you.

**Pull request merged**